scripts/scil_bundle_compute_endpoints_map.py

```diff
@@ -53,6 +53,9 @@ def _build_arg_parser():
     p.add_argument('--nb_points', type=int, default=1,
                    help="Number of points to consider at the extremities"
                         " of the streamlines. [%(default)s]")
+    p.add_argument('--mm', action='store_true',
+                   help='Compute the endpoints in mm instead of nb. of '
+                        'points. Useful for compressed streamlines.')
```

Comment:
I think this could be a little clearer. Like, if user adds `--mm`, but not --nb_points`, is it clear to them that it will take 1mm?

Maybe.... `If set, the value --nb_points will instead be understood as the length, in mm`. But I'm not sure that's perfect either.

Reply:
How about now ?